recipes/readline/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX || { cat config.log; exit 1; }
+make SHLIB_LIBS="$(pkg-config --libs ncurses)"
```

Comment:
That seems to be correct. Worrisome, but I guess there is nothing we can do about it.